src/env.h

```diff
@@ -1071,10 +1073,15 @@ class Environment : public MemoryRetainer {
 
   AliasedInt32Array stream_base_state_;
 
+  // Web Workers have a different time origin than the main thread. However,
+  // as PerformanceNodeTiming is exposed in worker_threads, we need to expose
+  // the per_process time origin of the main thread to the worker threads too.
   // https://w3c.github.io/hr-time/#dfn-time-origin
   uint64_t time_origin_;
   // https://w3c.github.io/hr-time/#dfn-get-time-origin-timestamp
   double time_origin_timestamp_;
+  // This is the time when the environment is created.
+  uint64_t environment_start_;
```

Comment:
Can these all be marked `const` + can the first two have a comment explaining that they are actually process-wide constants (and what the names of those are)?

Reply:
Thanks for the suggestion! Updated.